nautobot/dcim/filters.py

```diff
@@ -1722,3 +1726,27 @@ class Meta:
             "comments",
             "available_power",
         ]
+
+
+class ReliabilityInterfaceGroupFilterSet(BaseFilterSet, NameSlugSearchFilterSet):
+    """Filter for ReliabilityInterfaceGroup."""
+
+    class Meta:
+        """Meta attributes for filter."""
+
+        model = ReliabilityInterfaceGroup
+
+        # add any fields from the model that you would like to filter your searches by using those
+        fields = ["id", "name", "slug", "description", "members", "subscribers"]
+
+
+class ReliabilityInterfaceGroupAssociationFilterSet(BaseFilterSet, NameSlugSearchFilterSet):
+    """Filter for ReliabilityInterfaceGroupAssociation."""
+
+    class Meta:
+        """Meta attributes for filter."""
+
+        model = ReliabilityInterfaceGroupAssociation
+
+        # add any fields from the model that you would like to filter your searches by using those
+        fields = ["id", "group", "interface", "primary_ip", "standby_ip", "priority"]
```

Comment:
Since these are new models, is there any way we can use `fields = "__all__"` here?